sdk/runanywhere-python/AGENTS.md

```
@@ -0,0 +1,340 @@
+pytest tests/test_client.py
+pytest tests/test_client.py::test_context_manager_initializes_and_shuts_down -q
+```
+
+The tests prepend the package parent to `sys.path` themselves, so they run regardless of the
+invocation cwd. They monkeypatch `runanywhere._native.get_core` to return the fake and reset
+the process-wide native-lifecycle globals (`client._init_count`, `client._native_up`) around
+each test so ordering never leaks between tests.
+
+## Package Structure
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)